Add chi_poisson vs sky percentile plots

wtgee opened the pull request:
* Move the sky spectra reference plot to separate function.
* Add the chi_poisson plots to sky percentiles.
* Move the sky percentil plots to near the top.
* Adjust the colorbar slightly on the 2d chi plot.

wtgee committed:
Add chi_poisson vs sky percentile plots

* Move the sky spectra reference plot to separate function.
* Add the chi_poisson plots to sky percentiles.
* Move the sky percentil plots to near the top.
* Adding plot for all sky brightness.

kiyoyabe commented:
The order of arms looks like b -> n -> r here. Can you change to b -> r -> n? 
<img width="1881" alt="tmp" src="https://github.com/user-attachments/assets/7f166be5-a601-42a4-99b1-23290f2b3bbb" />

kiyoyabe commented:
Can you add collection name somewhere in the top page?

wtgee commented:
I've added the run name and fixed the arm order for the new plots. I've also rasterized the new plots as there are lots of data points.

Updated pdfs are in `u/wtg/tickets/PIPE2D-1716/01/plots` or you can generate again.

kiyoyabe commented:
A very minor comment: can we use "Sky Brightness Percentile" for y-axis label of p.3 plot too?
Also, is there any specific reason not to use the same layout for both p.2 (plot_vs_sky_brightness) and p.3 (plot_vs_sky_brightness_all)?

wtgee commented:
I made the two plots more consistent with each other by using the same plotting function. This handles the label as well as the style.
